src/components/Media/Media.tsx

```diff
@@ -32,6 +33,7 @@ export const Media = (props: MediaAllProps) => {
         previewImg,
         parallax = false,
         fullscreen,
+        fullscreenImageCover,
```

Comment:
If this is a boolean, we should name it "hasFullscreenCover" or something similar

Reply:
I think `isFullscreenImageCover` would be more appropriate than `has` prefix, isn't it?